packages/ui-extensions/src/surfaces/point-of-sale/targets.ts

```diff
@@ -20,6 +20,14 @@ export interface ExtensionTargets {
     StandardApi<'pos.home.modal.render'> & NavigationApi & ScannerApi,
     BasicComponents
   >;
+  'pos.purchase.post.action.render': RenderExtension<
```

Comment:
JS and I chose it on Friday because it matches what they used in this [spreadsheet](https://docs.google.com/spreadsheets/d/17kp4pbFOxgdF6XlmhtUi45tOtQL7maM5PQX8-PtjUjw/edit#gid=627583660) for the post purchase screen on unified extensions. There was some discussion about the name but they never resolved it. 

That said, I'm not attached to it. It's a type 2 decision until this is publicly released; we can easily change it.